### DISCUSSION TITLE
Vertex AI Quota Error generate_content_requests_per_minute_per_project_per_base_model?

### DISCUSSION THREAD

**Discussion by julianfalcionelli** [firebase/flutterfire#13294] [Q&A]
Hi everyone, I made a Flutter App with Vertex AI, and I constantly get a quota error ERROR Quota exceeded for `aiplatform.googleapis.com/generate_content_requests_per_minute_per_project_per_base_model`

This is happening when using Gemini Pro or Flash models.

My Google Cloud account is configured with Billing correctly, and when I go to the quotas section, the quota value for the `generate_content_requests_per_minute_per_project_per_base_model` service is at `1` as a maximum, and to ask for more quota they tell me that I have to contact the sales team, which I have already contacted repeatedly but I never receive a response.

<img width="1520" alt="4e19e3f012a87e4400adf524d3fbfbb9a7d455cf" src="https://github.com/user-attachments/assets/65b2ab07-c79a-487a-8408-52be62b3bd88">

I saw this happen to many people (I saw posts on Stackoverflow, Reddit) and I can’t find any solution.

I hope someone on the Google team will look into this.

I am using the latest vertex AI Plugin (`firebase_vertexai: ^0.2.3+1`).

Best.

**Comment by minhtaama** [comment]
Same issue here. Did you found any solution?
